brainstormforce/ultimate-addons-for-gutenberg
# Fix: "Conditional-block" Console Error when we Deactivate All blocks from the UAG dashboard

### Description
[Issue](https://share.getcloudapp.com/04uJ4kx9)

I have debugged this issue, so while deactivating all the blocks, using the "Deactivate All" button then the method "unregisterBlockType()" executed and we haven't registered uagb/conditional-block anywhere that's why we are getting this issue.

In this Conditional Block, we have generated hooks that are used in every block so we no need to add "uagb/conditional-block" block details in the config file.

### Test Cases:

1. Add any block in the editor.
2. Select the advanced tab and then select  Display Conditions.
3. Change the setting according to you and update the page.
4. Check whether it is working properly on the front-end or not.

### Types of changes
- Removed conditional block details from config file.

### How has this been tested?
Checked whether every display condition is working properly on the front-end or not

### Checklist:
